test/testcne/ring_test.c

```diff
@@ -108,6 +114,97 @@ ring_name_tests(void *args)
     return 0;
 }
 
+static int
+test_ring_init(void *args)
+{
+    unsigned int ring_size, i;
+    struct ring_test_info tst = {0};
+    ssize_t tsize;
+    uint32_t bufcnt, bufsz;
+    size_t sz;
+
+    CNE_SET_USED(args);
+
+    tst.tst   = tst_start("Ring init API");
+    ring_size = cne_align32pow2(rand() % 16384);
+    tst_ok("Using ring_size=%zu\n", ring_size);
+
+    tsize = cne_ring_get_memsize_elem(0, ring_size);
+    TST_ASSERT_AND_CLEANUP(tsize > 0, "Ring memsize < 0\n", test_ring_cleanup, &tst);
+
+    tst.mm = mmap_alloc((tsize / CNE_CACHE_LINE_SIZE), CNE_CACHE_LINE_SIZE, MMAP_HUGEPAGE_4KB);
+    TST_ASSERT_AND_CLEANUP(tst.mm != NULL, "Out of memory\n", test_ring_cleanup, &tst);
+
+    sz = mmap_size(tst.mm, &bufcnt, &bufsz);
+    CNE_INFO("tsize: %ld, mmap size %ld, bufcnt %d, bufsz %d\n", tsize, sz, bufcnt, bufsz);
```

Comment:
tsize - %zd
sz - %zu
bufcnt, bufsz - %u
would these format specifiers be better?